ansible/roles/shell/tasks/zsh.yml

```diff
@@ -44,3 +39,11 @@
     group: "{{ user.group }}"
     state: touch
   when: omzinstall is changed
+
+- name: Install zsh-autosuggestions
```

Comment:
`zsh-autosuggestions` is available via `community/zsh-autosuggestion`. Any reason not to use the package from the official sources? :-)

Reply:
good question I don't know why I did it this way :sweat_smile: I'll change it